src/20_Components/amp-call-tracking.html

```diff
@@ -40,11 +40,9 @@
 
     We also need to set a default phone number that will be shown before the endpoint responds. This is done via an `<a>` element inside the `amp-call-tracking` tag.
   -->
-  <p>
-    <strong>Call us!</strong><br>
+    <div>Call us!</div>
```

Comment:
Wondering: just we remove this as well? Or does it help to explain the context?

Reply:
removed